scripts/bdc/get_bdc_studies_md_from_picsure.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
+"""
+PicSure Data Extraction and Cleaning Tool
+
+This script extracts metadata from PicSure API based on user access to studies
+and cleans the extracted data according to specific criteria. To run this will need a PICSURE_TOKEN
+
+Usage:
+    python picsure_tool.py [--output-dir OUTPUT_DIR] [--token TOKEN]
+
+Requirements:
+    - PicSureClient
+    - PicSureBdcAdapter
+    - pandas
+    - numpy
+"""
+
+import os
+import sys
+import argparse
+import logging
+from pathlib import Path
+from datetime import datetime
+import pandas as pd
+import numpy as np
+
+try:
+    import PicSureClient
+    import PicSureBdcAdapter
+except ImportError as e:
+    sys.exit(f"Error: Could not import PicSure modules. {str(e)}")
+
+
+def clean_data(data):
+    """
+    Clean data based on specific criteria
+    """
+    df = data.copy()
+    
+    mask = (
+        df['dtId'].notna() &
+        df['varId'].notna() &
+        df['derived_var_name'].notna() &
+        df['description'].notna() &
+        df['varId'].str.startswith('phv')
+    )
+    
+    cleaned_df = df[mask]
+    cleaned_df = cleaned_df.reset_index(drop=True)
+    
+    return cleaned_df
+
+
+def extract_picsure_data(output_dir, token=None):
+    """
+    Extract metadata from PicSure API
+    """
+    log_file = output_dir / "picsure_extraction.log"
+    
+    logging.basicConfig(
+        level=logging.INFO,
+        format='%(asctime)s - %(levelname)s - %(message)s',
+        handlers=[
+            logging.FileHandler(str(log_file)),
+            logging.StreamHandler(sys.stdout)
+        ]
+    )
+    
+    output_file = output_dir / "pic_sure_data.csv"
+    
+    try:
+        logging.info("Starting PicSure data extraction")
+        
+        if token is None:
+            if "PICSURE_TOKEN" not in os.environ:
+                logging.error("PICSURE_TOKEN environment variable not set")
+                sys.exit("Error: PICSURE_TOKEN environment variable not set")
+            token = os.environ["PICSURE_TOKEN"]
+        
+        PICSURE_network_URL = "https://picsure.biodatacatalyst.nhlbi.nih.gov/picsure"
```

Comment:
```suggestion
        PICSURE_NETWORK_URL = "https://picsure.biodatacatalyst.nhlbi.nih.gov/picsure"
```

Constants in Python are usually all-caps (https://peps.python.org/pep-0008/#constants).